config/keys.js

```diff
@@ -1,4 +1,4 @@
 module.exports = {
-  PORT: process.env.PORT || 8888,
-  DATABASE: process.env.DB_STRING
-}
\ No newline at end of file
+  PORT: process.env.PORT || 3000,
```

Comment:
React uses port 3000 as default port, that comes with create-react-app configuration with webpack, so I really don't know how to change it. We should leave it to 8888 (or some other if you feel 8888 it's not suitable, but not 3000).